datalab/datalab_session/views.py

```diff
@@ -15,3 +18,24 @@ def get(self, request):
         for operation_clazz in operations.values():
             operation_details[operation_clazz.name()] = operation_clazz.wizard_description()
         return Response(operation_details)
+
+class AnalysisView(APIView):
+    """ 
+        View to handle analysis actions and return the results
+        To add a new analysis action, add a case to the switch statement and create a new file in the analysis directory
+    """
+    def post(self, request, action):
+        input = request.data
+
+        hdu = get_hdus(input['basename'])
```

Comment:
Again, probably just want to be able to get the SCI hdu by calling the function like

`get_hdu(input['basename'], 'SCI')`